### DISCUSSION TITLE
Suricata ERROR

### DISCUSSION THREAD

**Discussion by yokoulaaa** [Security-Onion-Solutions/securityonion#1621] [Unsupported Versions] [answered]
SO Version : 2.3.0
Install source : ISO Image
Install type: Eval 
Does so-status show all services running? Everything is running except suricata --> ERROR

All services are running well except Suricata when I use so-status. I tried to restart it and nothing changes. 
Then, I checked the /opt/so/log/suricata/suricata.log and I have these errors: 

![suricata log errors](https://user-images.githubusercontent.com/65095610/96975241-8040de80-151a-11eb-82b9-df4dd94cb58a.png)

I couldn't find suricata.yaml, but I modified defaults.yaml as suggested in the docs. I have changed nothing except: 
HOME_NET: "[192.168.0.0/16,10.0.0.0/8,172.16.0.0/12]" to HOME_NET: "[192.168.2.0/24]"

![defaults yaml](https://user-images.githubusercontent.com/65095610/96976207-d8c4ab80-151b-11eb-9da1-43226f7ae0cc.png)

Any insights on this ?

**Comment by weslambert** [comment, marked as answer]
What do you get by performing the following?

`docker exec -it so-suricata /bin/bash`
`more /etc/suricata/suricata.yaml`

Does the file look correct?  If not, try again restarting with:

`so-suricata-restart`

**Reply by yokoulaaa** [reply]
I have this output : 

![errors](https://user-images.githubusercontent.com/65095610/97238061-cd59d480-17e8-11eb-87f2-30ca94d179cb.png)

It looks like I have done something wrong during the installation process. But I don't know what.

**Reply by yokoulaaa** [reply]
I have done a full re-install and now it works :)
